src/Components/Character/CharacterInfoBar.css

```diff
@@ -1,27 +1,60 @@
-#characterInfoContainer {
-  --characterInfo-width: 400px;
+#characterInfoBarContainer {
+  --character-Image-size: 85px;
+  --characterInfo-width: 300px;
   --halfOf-characterInfo-width: calc(var(--characterInfo-width) / 2);
   position: absolute;
   left: calc(50% - var(--halfOf-characterInfo-width));
   top: 8px;
 }
 
-#characterInfo {
+#characterInfoBar {
   width: var(--characterInfo-width);
   height: 100px;
   background-color: darkblue;
   text-align: center;
   clip-path: polygon(
-    0 20px,
-    20px 0,
+    50px 20px,
+    70px 0,
     var(--characterInfo-width) 0,
     calc(var(--characterInfo-width) - 40px) 60px,
     180px 60px,
     160px 100px,
-    40px 100px
+    80px 100px
   );
 }
 
+
+.characterName {
+  float: right;
+  padding-right: 35px;
+  width: 100px;
+  font-family: 'Helvetica'; 
+	font-size: 20px;
+  	color: white;
+}
+
+.characterLevel {
+  padding-right: 30px;
+  padding-top: 35px;
```

Comment:
these hard-coded numbers are bad practice.
in this case - set a fixed size for the container, preferably via a css-var shared with the polygon.
inside the container - the texts divs can be aligned relatively.